libs/dao/src/main/java/com/akto/dao/ApiInfoDao.java

```diff
@@ -121,37 +124,18 @@ public Map<Integer,Integer> getLastTrafficSeen(){
         return result;
     }
 
-    public List<Bson> buildRiskScorePipeline(){
-        int oneMonthBefore = Context.now() - Constants.ONE_MONTH_TIMESTAMP;
-        String computedSeverityScore = "{'$cond':[{'$gte':['$severityScore',100]},2,{'$cond':[{'$gte':['$severityScore',10]},1,{'$cond':[{'$gt':['$severityScore',0]},0.5,0]}]}]}";
-        String computedAccessTypeScore = "{ '$cond': { 'if': { '$and': [ { '$gt': [ { '$size': '$apiAccessTypes' }, 0 ] }, { '$in': ['PUBLIC', '$apiAccessTypes'] } ] }, 'then': 1, 'else': 0 } }";
-        String computedLastSeenScore = "{ '$cond': [ { '$gte': ['$lastSeen', " +  oneMonthBefore + " ] }, 1, 0 ] }";
-        String computedIsSensitiveScore = "{ '$cond': [ { '$eq': ['$isSensitive', true] }, 1, 0 ] }";
-
-        List<Bson> pipeline = new ArrayList<>();
-        pipeline.add(Aggregates.project(
-            Projections.fields(
-                Projections.include("_id"),
-                Projections.computed("sensitiveScore",Document.parse(computedIsSensitiveScore)),
-                Projections.computed("isNewScore",Document.parse(computedLastSeenScore)),
-                Projections.computed("accessTypeScore",Document.parse(computedAccessTypeScore)),
-                Projections.computed("severityScore",Document.parse(computedSeverityScore))
-            )
-        ));
-
-        String computedRiskScore = "{ '$add': ['$sensitiveScore', '$isNewScore', '$accessTypeScore', '$severityScore']}";
-
-        pipeline.add(
-            Aggregates.project(
-                Projections.fields(
-                    Projections.include("_id"),
-                    Projections.computed("riskScore", Document.parse(computedRiskScore))
-                )
-            )
-        );
-        return pipeline;
+    public static Float getRiskScoreOfApiInfo(ApiInfo apiInfo){
```

Comment:
modify name to getRiskScore, and calculate everything in this function. For ex - risk score logic based on sensitive info should also be inside this

Reply:
done